src/index.html

```diff
@@ -5,8 +5,419 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__link-logo"
+      >
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#apple"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#samsung" class="header__link">Samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#smartphones" class="header__link">Smartphones</a>
+          </li>
+
+          <li
+            class="header__item"
+          >
+            <a
+              href="#laptops&computers"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#gadgets" class="header__link">Gadgets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#tablets" class="header__link">Tablets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#photo" class="header__link">Photo</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#video" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div class="card container__card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+          >
+
+          <h2 class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__rating">
+            <div class="card__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__revievs">Reviews: 5</p>
```

Comment:
```suggestion
            <p class="card__reviews">Reviews: 5</p>
```

Reply:
i dont understand what i must do in the 97 line